src/app/_services/authentication.service.ts

```diff
@@ -50,6 +52,18 @@ export class AuthenticationService {
       });
   }
 
+  joinChit(data): Observable<boolean> {
```

Comment:
move this to separate services file so that we can have all the joinchit related services at one place.